tests/page-objects/default/common/common.wdio.page.js

```diff
@@ -214,6 +220,13 @@ const syncWithoutWaitForSuccess = async () => {
   await (await syncButton()).click();
 };
 
+const syncAndWaitForFailure = async () => {
+  await openHamburgerMenu();
+  await (await syncButton()).click();
```

Comment:
```suggestion
  await syncWithoutWaitForSuccess();
```

Also could update line 197 in the same way.

Reply:
Unfortunately, the code on line 197 is used in an e2e test where the user gets logged out immediately after pressing the sync button, to the failed message is never displayed.

Reply:
Right. I'm saying we can reuse the `syncWithoutWaitForSuccess` function instead of duplicating it.